src/components/common/Modal.tsx

```
@@ -0,0 +1,163 @@
+    <div className="fixed inset-0 z-50 flex items-center justify-center px-4 py-6 sm:px-6">
+      <div
+        className="absolute inset-0 bg-black/40 backdrop-blur-[2px]"
+        onClick={onClose}
+      />
+
+      <div
+        role="dialog"
+        aria-modal="true"
+        aria-labelledby={title ? titleId : undefined}
+        className={cn(
+          'relative z-10 flex w-full max-h-[90vh] flex-col overflow-hidden rounded-3xl border border-border-base/80 bg-white shadow-[0_20px_55px_rgba(15,23,42,0.18)]',
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)